code/src/App.js

```diff
@@ -1,9 +1,41 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import { HappyThought } from './components/HappyThought';
+import { HappyForm } from './components/HappyForm';
+
+const url = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [postedMessage, setPostedMessage] = useState("")
+
+  useEffect(() => {
+    fetch(url)
+      .then(res => res.json())
+      .then(json => setThoughts(json))
+  }, [postedMessage])
+
+  const onFormSubmit = message => {
+    setPostedMessage(message)
+  }
+
+  const onLiked = thoughtId => {
+    console.log("Logging in the APP.js", thoughtId)
+  
+
```

Comment:
Maybe you can delete here the empty line, and everywhere else where it is not necessary to make your code more clean, otherwise very good job!